benchmarks/instruction_counts/execution/runner.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, min_core_id: int, max_core_id: int) -> None:
         self._min_core_id: int = min_core_id
         self._max_core_id: int = max_core_id
         self._num_cores = max_core_id - min_core_id + 1
+        print(f"Core pool created: {self._num_cores} cores")
```

Comment:
Maybe better to provide a bit more details here, I would like to know the concrete core ID list, for example:
```suggestion
        print(f"Core pool created: cores {self._min_core_id}-{self._max_core_id}")
```

Reply:
Done.